src/main/java/com/google/adwords/KenshooSelectorBuilder.java

```diff
@@ -0,0 +1,31 @@
+package com.google.adwords;
+
+import sync.google.KenshooSelector;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by user on 12/28/2016.
+ */
+public class KenshooSelectorBuilder {
+
+        private List<Integer> campaignIdInTargets;
+
+        public KenshooSelectorBuilder() {
+        }
+
+        public KenshooSelectorBuilder campaignIdInTargets(List<Integer> val) {
```

Comment:
instead of two methods for setting campaign ids, you can use one like this:
```
public Builder campaignIdInTargets(int... values) {
    this.campaignIdsInTarget = Seq.of(values).toList();
    return this;
}
```

and you could use it both ways:
```
builder.withCampaignIdsInTarget(1);

or

builder.withCampaignIdsInTarget(1, 2, 3);
```